src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -448,8 +447,20 @@ private void updateLevelButton()
      */
     private void confirmClicked(Button button)
     {
-        MineColonies.getNetwork().sendToServer(new BuildToolPlaceMessage(hutDec.get(hutDecIndex),
-                                                                          getStyles().get(styleIndex), this.pos, rotation, Settings.instance.isInHutMode()));
+        InventoryPlayer inventory = this.mc.thePlayer.inventory;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'inventory' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)